pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -2243,6 +2243,176 @@ def is_hermitian(self):
         return True
 
 
+class IntegerComparator(Operation):
+    r"""IntegerComparator(value, geq, control_wires, wires)
+    Apply a Pauli X gate controlled on the following condition:
```

Comment:
I suggest making this sentence stand-alone. Keen users can read further to check what the condition is about exactly. (Suggestion can be improved).
```suggestion
    Apply a controlled Pauli X gate using integer comparison as the condition.
```